style/variables.less

```diff
@@ -12,13 +10,4 @@
     color: @color;
     opacity: 1;
   }
-
-  // IE
-  &:-ms-input-placeholder {
-    color: @color;
-  }
-  // Edge
-  &::-ms-input-placeholder {
-    color: @color;
-  }
```

Comment:
Since canary won't be used for a while still, I figure we may as well remove those now.